src/components/SigningModals/TransactionStepper/Step/Illustration.js

```diff
@@ -0,0 +1,101 @@
+import React, { useCallback } from 'react'
+import PropTypes from 'prop-types'
+import { STEP_ERROR, STEP_SUCCESS, STEP_WORKING } from '../stepper-statuses'
+
+const gradients = {
+  [STEP_WORKING]: [
+    ['#98f9ff', '#00c2ff'],
+    ['#01e8f7', '#00c2ff'],
+  ],
+  [STEP_SUCCESS]: [
+    ['#9fe8d2', '#00c2ff'],
+    ['#73d8ce', '#18b9a0'],
+  ],
+  [STEP_ERROR]: [
+    ['#ffa48e', '#f8baab'],
+    ['#fa8e75', '#ef7757'],
+  ],
+}
+
+function Illustration({ status }) {
+  const [backgroundFrom, backgroundTo] = gradients[status][0]
+  const [foregroundFrom, foregroundTo] = gradients[status][1]
+
+  const gradientId = useCallback(id => `${status.description}-${id}`, [status])
```

Comment:
Inline SVGs don't scope their defs so we have to provide distinct ids, here i'm using the symbol `description` to assign the colours correctly.

Reply:
Yes, if you have a green gradient and a red gradient assigned with the same id, the first one rendered in source order will be assigned to the matching shape (SVG painting is the reverse of typical dom z-index assignment). So if red and green have the same id but green is rendered first in the document, then all shapes in the document assigned to that id will be green regardless.